personal_notion_bot/notion_side/send_page.py

```diff
@@ -0,0 +1,103 @@
+import requests
+import json
+import sys; sys.path.append("..")
```

Comment:
Using sys.path.append is ok if you test something locally, but not so good in production code. 

The best practice here is to make every project directory a module by adding `__init__.py` file in every directory (personal_notion_bot, notion_side, telegram_side, ...). Then use `from personal_notion_bot.config import HEADERS`

Same with other files. 

To access locally `person_notion_bot` you will need to add the root directory containing `personal_notion_bot` to `PYTHONPATH` environment variable. P